### ISSUE TITLE
Undeclared purepng > 0.1 dependency

### ISSUE THREAD

**Issue by moreati** [brechtm/rinohtype#7]
`setup.py` should declare the dependency on `purepng`, in `install_requires`. Note that currently this dependency is on an unreleased version of purepng.

Running rinoh in a fresh virtualenv. :
```
(env) alex@martha:~/src/rinohtype$ pip install -e .
...
Successfully installed RinohType
Cleaning up...
(env) alex@martha:~/src/rinohtype$ rinoh
Traceback (most recent call last):
  File "/home/alex/src/rinohtype/env/bin/rinoh", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/home/alex/src/rinohtype/bin/rinoh", line 4, in <module>
    from rinoh.tool import main
  File "/home/alex/src/rinohtype/rinoh/__init__.py", line 37, in <module>
    module = import_module('.' + name, __name__)
  File "/usr/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/home/alex/src/rinohtype/rinoh/document.py", line 31, in <module>
    from .backend import pdf
  File "/home/alex/src/rinohtype/rinoh/backend/pdf/__init__.py", line 19, in <module>
    from .png import PNGReader
  File "/home/alex/src/rinohtype/rinoh/backend/pdf/png.py", line 11, in <module>
    import png as purepng
ImportError: No module named 'png'
```

Install the latest released version of purepng (0.1.0), running `rinoh` raises an AttributeError:
```
(env) alex@martha:~/src/rinohtype$ pip install purepng
Downloading/unpacking purepng
  Downloading purepng-0.1.0.zip (284kB): 284kB downloaded
...
Successfully installed purepng
Cleaning up...
(env) alex@martha:~/src/rinohtype$ rinoh
Traceback (most recent call last):
  File "/home/alex/src/rinohtype/env/bin/rinoh", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/home/alex/src/rinohtype/bin/rinoh", line 4, in <module>
    from rinoh.tool import main
  File "/home/alex/src/rinohtype/rinoh/__init__.py", line 37, in <module>
    module = import_module('.' + name, __name__)
  File "/usr/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/home/alex/src/rinohtype/rinoh/document.py", line 31, in <module>
    from .backend import pdf
  File "/home/alex/src/rinohtype/rinoh/backend/pdf/__init__.py", line 19, in <module>
    from .png import PNGReader
  File "/home/alex/src/rinohtype/rinoh/backend/pdf/png.py", line 166, in <module>
    RENDERING_INTENT = {purepng.ABSOLUTE_COLORIMETRIC: ABSOLUTE_COLORIMETRIC,
AttributeError: 'module' object has no attribute 'ABSOLUTE_COLORIMETRIC'
```

Install an unreleased build of purepng, straight from github. `rinoh` runs
```
(env) alex@martha:~/src/rinohtype$ pip uninstall purepng
Uninstalling purepng:
...
  Successfully uninstalled purepng
(env) alex@martha:~/src/rinohtype$ pip install https://github.com/Scondo/purepng/archive/58f553e62a4b608df04b19d2ff83f12607b49f63.zip
Downloading/unpacking https://github.com/Scondo/purepng/archive/58f553e62a4b608df04b19d2ff83f12607b49f63.zip
  Downloading 58f553e62a4b608df04b19d2ff83f12607b49f63.zip (unknown size): 267kB downloaded
  Running setup.py (path:/tmp/pip-tdvayijc-build/setup.py) egg_info for package from https://github.com/Scondo/purepng/archive/58f553e62a4b608df04b19d2ff83f12607b49f63.zip
    
    warning: no directories found matching 'html'
    warning: no previously-included files matching '*.pyc' found anywhere in distribution
    warning: no previously-included files matching '*.DS_Store' found anywhere in distribution
Installing collected packages: purepng
  Running setup.py install for purepng
    
    warning: no directories found matching 'html'
    warning: no previously-included files matching '*.pyc' found anywhere in distribution
    warning: no previously-included files matching '*.DS_Store' found anywhere in distribution
Successfully installed purepng
Cleaning up...
(env) alex@martha:~/src/rinohtype$ rinoh
usage: rinoh [-h] [--paper [PAPER]] [input]

Render a reStructuredText document to PDF.

positional arguments:
  input            the reStructuredText document to render

optional arguments:
  -h, --help       show this help message and exit
  --paper [PAPER]  the paper size to render to (default: A4)
(env) alex@martha:~/src/rinohtype$ 
```

**Comment by brechtm** [comment]
Fixed in e9b5787

**Comment by brechtm** [comment]
Fixed when installing from sources. Still broken installing from wheel.

**Comment by brechtm** [comment]
Fixed in fbcf39a
